packages/components/src/components/hds/form/label/index.hbs

```diff
@@ -5,4 +5,7 @@
 <label class={{this.classNames}} for="{{@controlId}}" id={{this.id}} ...attributes>
   {{yield}}
   <Hds::Form::Indicator @isRequired={{@isRequired}} @isOptional={{@isOptional}} />
+  <span class="sr-only">
+    {{@hiddenText}}
+  </span>
```

Comment:
[suggestion] probably better to add a guard, to avoid adding DOM nodes when not needed
```suggestion
  {{#if @hiddenText}}
    <span class="sr-only">
      {{@hiddenText}}
    </span>
  {{/if}}
```